<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    font-family: 'Oswald', sans-serif;
    font-weight: 400;
}

QMenuBar{
    background-color: #293045;
    padding: 0;    
    font-size: 14px;
    color:white;
}

QMenuBar:item{
    padding: 17px;
    border-radius: 10px;

}

QMenuBar:item:selected{
    background-color: #768AC5;
    color:black;
    border-radius: 5px;
    margin: 10px;
    
    

}

QMenu{
    background-color: #293045;

    color: white;
}

QMenu:item{
    padding: 5px;
    border-radius: 5px;
}

QMenu:item:selected{
    background-color: #768AC5;
    color:white;

}
Qmenu#icon{
   icon:size 40px; 
    
}


QRadioButton:indicator{
    border-style: solid;
    border-color:  #293045;
    border-radius: 6%;
    border-width:  2px;
    background-color: #BABFCE;
}
QRadioButton::indicator:checked{
    background-color: #768AC5;
}

#label{
    font-size: 35px;
}

#usuariosLabel{
    font-size: 14px;
}

#contraseALabel{
    font-size: 14px;
}

#usuariosLineEdit{
    width: 170px;
    height: 20px;
    border-radius: 10px;
}

#passwordLineEdit{
    width: 170px;
    height: 20px;
    border-radius: 10px;
}

#labelprofesores{
    font-size: 26px;
}

QLineEdit{
    border-radius: 5px;
    background-color: #BABFCE;
    
}

#title{
    width: 100px;
    height: 200px;
}



QPushButton{
    border-color:#293045;
    border-style: solid;
    border-width: 1px;
    border-radius: 6px;
    background-color: #293045;
    color:#ffffff;
    height: 30px;
    padding: 0px 4px;

}
QPushButton:hover{
    background-color:#768AC5;
    color:#293045;
}
QPushButton:pressed {
    background-color: #293045;

}

#pushButton_5{
    padding: 10px;
    background-color: #768AC5;
    border-radius: 15%;
}

#pushButton_2, #pushButton_3, #pushButton_4{
    height: 30px;
}
#pushButton_5 {
	width: 10px;
	height: 10px
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>771</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labelprofesores">
           <property name="text">
            <string>Profesores</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>721</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <item row="1" column="0">
            <property name="text">
             <string>aaaaa</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>aaaa</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>aaaaa</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>aaaa</string>
            </property>
           </item>
           <item row="5" column="0">
            <property name="text">
             <string>aaaaa</string>
            </property>
           </item>
           <item row="6" column="0">
            <property name="text">
             <string>aaaaaa</string>
            </property>
           </item>
           <item row="7" column="0">
            <property name="text">
             <string>aaaaaa</string>
            </property>
           </item>
           <item row="8" column="0">
            <property name="text">
             <string>aaaaaa</string>
            </property>
           </item>
           <item row="9" column="0">
            <property name="text">
             <string>aaaaa</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>291</width>
          <height>54</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../blustock/lupa.png</normaloff>../blustock/lupa.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="cursorPosition">
            <number>0</number>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string notr="true"/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>60</y>
          <width>401</width>
          <height>34</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit"/>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Aceptar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>410</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>CONFIRMAR</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>410</y>
          <width>281</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Borrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Crear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Editar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>56</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>HERRAMIENTAS</string>
    </property>
    <addaction name="actionGesti_n_de_subgrupos"/>
    <addaction name="actionGesti_n_de_grupos"/>
    <addaction name="actionListado"/>
    <addaction name="actionConsultas"/>
   </widget>
   <widget class="QMenu" name="menuPersonal">
    <property name="title">
     <string>PERSONAL</string>
    </property>
    <addaction name="actionAlumnos"/>
    <addaction name="actionProfesores"/>
   </widget>
   <widget class="QMenu" name="menuMovimientos">
    <property name="title">
     <string>MOVIMIENTOS</string>
    </property>
    <addaction name="actionGestion_de_movimientos"/>
   </widget>
   <widget class="QMenu" name="menuProfesor">
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>PROFESOR</string>
    </property>
   </widget>
   <addaction name="menuHerramientas"/>
   <addaction name="menuPersonal"/>
   <addaction name="menuMovimientos"/>
   <addaction name="menuProfesor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGesti_n_de_subgrupos">
   <property name="text">
    <string>Gestión de subgrupos</string>
   </property>
  </action>
  <action name="actionGesti_n_de_grupos">
   <property name="text">
    <string>Gestión de grupos</string>
   </property>
  </action>
  <action name="actionListado">
   <property name="text">
    <string>Listado</string>
   </property>
  </action>
  <action name="actionConsultas">
   <property name="text">
    <string>Consultas</string>
   </property>
  </action>
  <action name="actionGestion_de_movimientos">
   <property name="text">
    <string>Gestion de movimientos</string>
   </property>
  </action>
  <action name="actionAlumnos">
   <property name="text">
    <string>Alumnos</string>
   </property>
  </action>
  <action name="actionProfesores">
   <property name="text">
    <string>Profesores</string>
   </property>
  </action>
  <action name="actionlogo">
   <property name="text">
    <string>logo</string>
   </property>
  </action>
  <action name="actionaa">
   <property name="text">
    <string>aa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
